src/main/java/baseball/service/ComputerService.java

```diff
@@ -0,0 +1,22 @@
+package baseball.service;
+
+import baseball.model.BaseballNumber;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ComputerService {
+    public List<Integer> generateBaseballNumber() {
+        return IntStream.generate(() -> Randoms.pickNumberInRange(1, 9))
+                .distinct()
+                .limit(3)
+                .boxed()
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```

함수형 프로그래밍의 Stream을 통해 Collection 객체를 생성할 때, 자바17부터 사용 가능한 toList()를 사용하면 unmodifiableList로 return해주기 때문에return된 값을 안전하게 관리할 수 있습니다!

Reply:
오오 배워갑니다 !!! 감사합니다 !! 짱짱 !